src/EventManager.php

```diff
@@ -54,20 +64,25 @@ class EventManager implements EventManagerInterface
      *
      * @param  null|string|int|array|Traversable $identifiers
```

Comment:
I'm not a fan of such permissive function. Can't we agree on only accepting array so that we can use array $identifiers = [] ?